xml/2018-11-25.xml

```diff
@@ -0,0 +1,245 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="Weekly Digest">
+
+  <image
+    width="80"
+    src="https://i.imgur.com/NwTWnqP.jpg"
+    alt="Stack of biscuits on a plate"
+    title="Oddly satisfying" />
+
+  <hr/>
+
+  <text>
+    Hi everyone!
+
+    The past week week was another fun week, with ice skating sessions by the
+    Natural History Museum and a great security talk from Facebook.
+
+    In the coming week, we have another social trip (to watch The Book of Mormon),
+    followed by two major events: our joint diversity panel event with CGCU on
+    Thursday and Google DevFest London on Saturday. Also, Liquidnet have an open
+    office on Tuesday, involving casual networking drinks open to all DoCSoc
+    members, so if you're looking for a relaxed evening in the City of London,
+    this would be a good opportunity to take. Read on for more information!
+  </text>
+
+  <hr />
+
+  <text>## General Notices</text>
+
+  <hr />
+
+  <heading>DevFest Volunteers</heading>
+  <text>
+    DevFest is _this_ Saturday (1st December)! The event is already sold out, with
+    1000 people attending. We need as many hands as possible for this event so
+    if you're interested in helping out, join the [DoCSoc: Operations](https://www.facebook.com/groups/1509850436007222/)
+    group.
+  </text>
+
+  <hr />
+
+  <heading>Careers Service: CV Reviews</heading>
+  <text>
+    Looking to get your CV reviewed? The Careers Service has an online review
+    system, My CVs, where you can upload your CV, provide details and get feedback
+    from the Careers team within three working days.
+
+    To get started, upload your CV under the "My Profile" section of [JobsLive](http://www.imperial.ac.uk/careers/services/jobslive/).
+  </text>
```

Comment:
Add details for final Friday Doc Carreers Talk of the year: Friday 1PM-1:50PM- Making the most of LinkedIn to promote your personal brand and how to network effectively.